code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -65,6 +65,23 @@
 	else
 		return ..()
 
+/obj/structure/cult/functional/proc/updatehealth()
+	if(health <= 0)
+		destroy_structure()
+
+/obj/structure/cult/functional/proc/take_damage(damage, damage_type = BRUTE)
+	if(damage_type != STAMINA)
```

Comment:
only `BRUTE` and `BURN` should work.

I don't think an oxygen damaging or clone damaging bullet should destroy this.